code/modules/atmospherics/gasmixtures/gas_types.dm

```diff
@@ -77,22 +77,34 @@ GLOBAL_LIST_INIT(hardcoded_gases, list("o2","n2","co2","plasma")) //the main fou
 	moles_visible = 1
 	dangerous = TRUE
 
-/datum/gas/oxygen_agent_b
-	id = "agent_b"
-	specific_heat = 300
-	name = "Oxygen Agent B"
+/datum/gas/brown_gas //This is nitric oxide, but given generic name to avoid confusion with nitrous oxide(N20 vs. NO2)
+	id = "browns"
+	specific_heat = 20
+	name = "Brown Gas"
+	dangerous = TRUE
 
-/datum/gas/volatile_fuel
```

Comment:
You need to change gas miners so they don't mine this stuff anymore, https://github.com/tgstation/tgstation/blob/c5fce5b01ec4aed6812f4f8d1586a09bc22e5e5f/code/modules/atmospherics/machinery/other/miner.dm#L181